<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scriitori</class>
 <widget class="QWidget" name="Scriitori">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scriitori</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="desEdit"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTableView" name="ScriitoritableView"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="reviseButton">
     <property name="text">
      <string>Revise</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="AddButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Act</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="ActEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="plotButton">
     <property name="text">
      <string>Save Plot</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
